docs/installation.rst

```diff
@@ -64,7 +96,7 @@ You can install tools and utilities using following syntax;
 .. code-block:: bash
 
     python -m aqt [-h][--help][-O | --outputdir <directory>][-b | --base <mirror url>][-E | --external <7zip command>] \
-        tool <host> <tool_name> <tool-version> <arch> [--timeout <timeout>]
+        install-tool <host> <tool_name> <tool-version> <arch> [--timeout <timeout>]
```

Comment:
I thought that `<tool-version>` was no longer an argument for this tool. I can't see any evidence of it in `Cli._set_install_tool_parser`.